### PR TITLE
fix some typos in 0.6.0 release notes

### PR DESCRIPTION
**Pull request by dbandstra**
I tried to read through the whole thing. Please double check that I didn't break the intending meaning of anything. Some of these changes are more anal than others. Speaking of which, "merge_anal_dumps"... yeesh.

Also, I liked the Zen programming language link.

### CODE REVIEW

File: src/download/0.6.0/release-notes.html [hunk 1]
```diff
@@ -2444,7 +2444,7 @@ <h1>0.6.0 Release Notes</h1>
     _ = 0xff00_00ff;
     _ = 0b10000000_10101010;
     _ = 0b1000_0000_1010_1010;
-    _ = 0x123_190.109_038_018p102;
+    _ = 0x123_190.109_038_018_102;
```
Comment (dbandstra) [inline src/download/0.6.0/release-notes.html:2447]: I assumed the "p" was a typo

Comment (mikdusan) [reply src/download/0.6.0/release-notes.html:2447]: I think it's legit. That literal came from https://github.com/ziglang/zig/issues/504 and `docgen` release-notes actually runs that code and it passes. https://en.cppreference.com/w/cpp/language/floating_literal also shows the same syntax.

Comment (dbandstra) [reply src/download/0.6.0/release-notes.html]: Ok, I reverted it.

**Comment by andrewrk** [comment]
Thanks! Will upload in a few minutes

**Pull request merged**